DynamicSize/DynamicSizeSnapshot.swift

```diff
@@ -164,13 +166,14 @@ func performDynamicSizeSnapshotTest(_ name: String?, sizes: [String: CGSize], is
     let result = sizes.map { (sizeName, size) -> Bool in
         // swiftlint:disable:next force_unwrapping
         let view = instance.snapshotObject!
+        let _snapshotName = identifier != nil ? "\(snapshotName)_\(identifier!) - \(sizeName)" : "\(snapshotName) - \(sizeName)"
 
         resizer.resize(view: view, for: size)
 
         return FBSnapshotTest.compareSnapshot(instance, isDeviceAgnostic: isDeviceAgnostic, usesDrawRect: usesDrawRect,
-                                              snapshot: "\(snapshotName) - \(sizeName)", record: isRecord,
+                                              snapshot: _snapshotName, record: isRecord,
                                               referenceDirectory: referenceImageDirectory, tolerance: tolerance,
-                                              filename: actualExpression.location.file)
+                                              filename: actualExpression.location.file, identifier: nil)
```

Comment:
Should we be passing in the `identifier` here?

Reply:
No is good here. As we are using the method in the library with the `selector` and we are modifying the name already here passing the `identifier` doesn't work well. That is why we I added the` identifier` before the `sizeName` and passed `nil` as an argument to avoid the problem.